sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -269,6 +273,7 @@ async Task OnBreakPointHit (JObject args, CancellationToken token)
 						var method_token = mono_frame ["method_token"].Value<int> ();
 						var assembly_name = mono_frame ["assembly_name"].Value<string> ();
 
+						Console.WriteLine ($"frame pos {il_pos} method_token {method_token} asm {assembly_name}");
```

Comment:
will we always have an il_pos, etc here? Is it possible someone would invoke fire_bp directly from JS? I can imagine wanting to do it but maybe it's not necessary to support that scenario. (It's unclear to me here whether frames in this case is all stack frames or just mono stack frames, so ignore me if it's the latter)

Reply:
Users invoking the JS internals for debugger support is undefined behavior